src/sentry/integrations/messaging/__init__.py

```diff
@@ -5,3 +5,5 @@
 TODO: Move the packages for individual integrations to be subpackages of this one
       (possibly in coordination with grouping other integrations by purpose)
 """
+
+from .message_builder import *  # noqa: F401,F403
```

Comment:
is this for a circular import? 👀  Like how much do we need this, 'cause I think __init__ files that export modules with a bunch of stuff will cause more circular imports down the line.

Reply:
yes this is for a circular import